ext/dba/dba.h

```diff
@@ -143,6 +175,88 @@ extern int dba_set_timer( DBA_TID tid, size_t seconds );
 /// Return 0 on success, otherwise -1 is returned and the dba_errno is set
 extern int dba_set_sample( DBA_TID tid, const char* path );
 
+/// Enable block tracer in kernel address for the DBA task speicified by ID
+/// Note that the task must be IDLE to be configurable
+///
+///     \param  tid     DBA task ID
+///
+/// Return 0 on success, otherwise -1 is returned and the dba_errno is set
+extern int dba_enable_block_tracer_is_kernel( DBA_TID tid );
+
+/// Disable block tracer in kernel address for the DBA task speicified by ID
+/// Note that the task must be IDLE to be configurable
+///
+///     \param  tid     DBA task ID
+///
+/// Return 0 on success, otherwise -1 is returned and the dba_errno is set
+extern int dba_disable_block_tracer_is_kernel( DBA_TID tid );
+
+/// Enable block tracer for the DBA task speicified by ID
+/// Note that the task must be IDLE to be configurable
+///
+///     \param  tid     DBA task ID
+///
+/// Return 0 on success, otherwise -1 is returned and the dba_errno is set
+extern int dba_enable_block_tracer( DBA_TID tid );
+
+/// Disable block tracer for the DBA task speicified by ID
+/// Note that the task must be IDLE to be configurable
+///
+///     \param  tid     DBA task ID
+///
+/// Return 0 on success, otherwise -1 is returned and the dba_errno is set
+extern int dba_disable_block_tracer( DBA_TID tid );
+
+/// Enable instruction tracer in kernel address for the DBA task speicified by ID
+/// Note that the task must be IDLE to be configurable
+///
+///     \param  tid     DBA task ID
+///
+/// Return 0 on success, otherwise -1 is returned and the dba_errno is set
+extern int dba_enable_instr_tracer_is_kernel( DBA_TID tid );
+
+/// Disable instruction tracer in kernel address for the DBA task speicified by ID
+/// Note that the task must be IDLE to be configurable
+///
+///     \param  tid     DBA task ID
+///
+/// Return 0 on success, otherwise -1 is returned and the dba_errno is set
+extern int dba_disable_instr_tracer_is_kernel( DBA_TID tid );
+
+/// Enable instruction tracer for the DBA task speicified by ID
+/// Note that the task must be IDLE to be configurable
+///
+///     \param  tid     DBA task ID
+///
+/// Return 0 on success, otherwise -1 is returned and the dba_errno is set
+extern int dba_enable_instr_tracer( DBA_TID tid );
+
+/// Disable instruction tracer for the DBA task speicified by ID
+/// Note that the task must be IDLE to be configurable
+///
+///     \param  tid     DBA task ID
+///
+/// Return 0 on success, otherwise -1 is returned and the dba_errno is set
+extern int dba_disable_instr_tracer( DBA_TID tid );
+
+/// Set the global variable
+/// Note that the task must be IDLE to be configurable
+///
+///     \param  tid     DBA task ID
+///     \param  path    path to the global variable json file
+///
+/// Return 0 on success, otherwise -1 is returned 
+extern int dba_set_global( DBA_TID tid, const char* path );
```

Comment:
Make the API naming more clear. 
e.g. `dba_set_globalvar_json( DBA_TID tid, const char* json_fpath )`
The "set_global" can be ambiguous.